src/utils/numberUtils.js

```diff
@@ -0,0 +1,13 @@
+export const isAllNumber = (arr) => {
+  return arr.every((item) => !isNaN(item));
+};
+
+export const isAllNumberPositive = (arr) => {
+  let IS_POSITIVE = arr.every((item) => item >= 0);
+
+  return IS_POSITIVE;
```

Comment:
여기서는 let보다는 const를 사용하시는게 좋지 않을까요? 변수명도 스네이크케이스를 사용하셔서 상수인 것 같아요!
추가로, 이렇게 축약할수도 있을 것 같습니다!
```suggestion
  return arr.every((item) => item >= 0);
```

Reply:
좋아요~!